src/components/Login/Login.js

```diff
@@ -20,8 +20,7 @@ export const Login = props => {
   const handleSubmit = e => {
     e.preventDefault();
     if (validateSubmit()) {
-      const user = {username, password};
-      props.setUser(user);
+      props.setUser(username);
```

Comment:
In order to have the logged in user persist across pages/refresh, I wonder if we need to hold it in App and pass it as a prop to each component (even though its being set/updated in Login).  Similar to how in Ideabox we deleted an idea within a component but it sent it back and updated the state of App?

It may be worth back-burnering the concerns about the user as an issue/ticket for now since its not a key focus of the project requirements and coming back to it if we have time later?